.github/workflows/bungee-asan.yml

```
@@ -0,0 +1,188 @@
+          -DMAD=OFF
+          -DMODPLUG=OFF
+          -DWAVPACK=OFF
+          -DFFMPEG=OFF
+          -DDOWNLOAD_MANUAL=OFF
+          -DINSTALL_USER_UDEV_RULES=OFF
+          -DCMAKE_C_COMPILER_LAUNCHER=ccache
+          -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+
+      - name: "Build mixxx-test"
+        working-directory: build
+        run: cmake --build . --target mixxx-test -j$(nproc)
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:39: Quote this to prevent word splitting

(shellcheck)